<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>665</width>
    <height>622</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Port:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="combPort"/>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>BaudRate:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditBaudRate"/>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>DataBits:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="combDataBits"/>
     </item>
     <item>
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>WrapCols:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="combParity"/>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>StopBits:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="combStopBits"/>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Parity:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="combWrapCols"/>
     </item>
     <item>
      <widget class="QPushButton" name="bntReflush">
       <property name="text">
        <string>Reflush</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnOpenClose">
       <property name="text">
        <string>Open</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_6">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTextEdit" name="textEditRx">
     <property name="font">
      <font>
       <family>YaHei Consolas Hybrid</family>
       <pointsize>12</pointsize>
      </font>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="textEditTx">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>160</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>YaHei Consolas Hybrid</family>
       <pointsize>12</pointsize>
      </font>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="btnSend">
       <property name="text">
        <string>Send</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnSendFile">
       <property name="text">
        <string>SendFile</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnClearRx">
       <property name="text">
        <string>ClearRx</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelTxRxCnt">
       <property name="text">
        <string>TextLabel</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLineEdit" name="lineEditMs"/>
     </item>
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>ms</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="chkHexMode">
       <property name="text">
        <string>HexMode</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="chkAutoRepeat">
       <property name="text">
        <string>AutoRepeat</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="chkAutoClear">
       <property name="text">
        <string>AutoClear</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>btnOpenClose</sender>
   <signal>clicked()</signal>
   <receiver>Widget</receiver>
   <slot>slotOpenClose()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>810</x>
     <y>24</y>
    </hint>
    <hint type="destinationlabel">
     <x>431</x>
     <y>311</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnSend</sender>
   <signal>clicked()</signal>
   <receiver>Widget</receiver>
   <slot>slotSendData()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>52</x>
     <y>598</y>
    </hint>
    <hint type="destinationlabel">
     <x>431</x>
     <y>311</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkAutoRepeat</sender>
   <signal>clicked()</signal>
   <receiver>Widget</receiver>
   <slot>slotAutoRepeat()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>799</x>
     <y>597</y>
    </hint>
    <hint type="destinationlabel">
     <x>431</x>
     <y>311</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnClearRx</sender>
   <signal>clicked()</signal>
   <receiver>Widget</receiver>
   <slot>slotClearRx()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>135</x>
     <y>600</y>
    </hint>
    <hint type="destinationlabel">
     <x>477</x>
     <y>311</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnSendFile</sender>
   <signal>clicked()</signal>
   <receiver>Widget</receiver>
   <slot>slotSendFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>135</x>
     <y>600</y>
    </hint>
    <hint type="destinationlabel">
     <x>477</x>
     <y>311</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bntReflush</sender>
   <signal>clicked()</signal>
   <receiver>Widget</receiver>
   <slot>slotReflush()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>836</x>
     <y>24</y>
    </hint>
    <hint type="destinationlabel">
     <x>480</x>
     <y>311</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>slotOpenClose()</slot>
  <slot>slotSendData()</slot>
  <slot>slotAutoRepeat()</slot>
  <slot>slotClearRx()</slot>
  <slot>slotSendFile()</slot>
  <slot>slotReflush()</slot>
 </slots>
</ui>
